misc/exec-command-agent-test/kill/go.mod

```diff
@@ -0,0 +1,3 @@
+module kill
```

Comment:
Why exactly are go mod files being added here?

Reply:
they are required for building with go 1.17+ unless you set the GO111MODULE=auto env var.